src/installer/managed/Microsoft.NET.HostModel/Microsoft.NET.HostModel.csproj

```diff
@@ -4,18 +4,21 @@
     <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
IMO, the intervening `managed/` directory could be deleted as there is only one project left.
(or better we just move `Microsoft.NET.HostModel` to its canonically sound destination; under src/libraries 😁)

Reply:
Yes, `Microsoft.NET.HostModel` will be moved to libraries. I didn't move all these files to avoid the git changes and loosing the history in the WebUI as we will anyway move it at some point in the NET6 cycle :)

That's actually an up-for-grabs task if you would be interested ;)